pkg/cmd/experimental/diagnostics/master.go

```diff
@@ -0,0 +1,143 @@
+package diagnostics
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	diagnosticflags "github.com/openshift/origin/pkg/cmd/experimental/diagnostics/options"
+	"github.com/openshift/origin/pkg/diagnostics/log"
+	masterdiagnostics "github.com/openshift/origin/pkg/diagnostics/master"
+	systemddiagnostics "github.com/openshift/origin/pkg/diagnostics/systemd"
+	diagnostictypes "github.com/openshift/origin/pkg/diagnostics/types/diagnostic"
+)
+
+const (
+	MasterDiagnosticsRecommendedName = "master"
+
+	StandardMasterConfigPath string = "/etc/openshift/master/master-config.yaml"
+)
+
+var (
+	AvailableMasterDiagnostics = util.NewStringSet("AnalyzeLogs", "UnitStatus", "MasterConfigCheck")
```

Comment:
I don't like having a list of string literals that have to match something somewhere else. If it has to be done, should be shared constants, but that's silly overkill here. What was wrong with putting all available diagnostics in a map[string]Diagnostic?